src/style.css

```diff
@@ -1,3 +1,179 @@
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --white: #fff;
+  --lightgrey: #f3f3f3;
+  --darkgrey: #060b35;
+  --grey: #616070;
+  --blue: #00acdc;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  justify-content: center;
+}
+
+.card__rectangle {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  width: 198px;
+  height: 406px;
+  flex-shrink: 0;
+
+  border-radius: 5px;
+  border: 1px solid var(--lightgrey);
+  background: var(--white);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  margin: 32px 19px 0;
+}
+
+.card__description {
+  color: var(--darkgrey);
+  font-family: Roboto, sans-serif;
```

Comment:
You can set font family and default font styles for the body tag to have it in one place and to get elements inherited by it